PutsyloMatvii/HW11/1.py

```diff
@@ -0,0 +1,14 @@
+def check_age():
```

Comment:
look at this version of code:
def validate_age(age):
    if age <= 0 :
        raise ValueError("Only positive integers are allowed")
 
    if age % 2 == 0:
        print("age is even")
    else:
        print("age is odd")

# #################################### 
try:
    num = int(input("Enter your age: "))
    validate_age(num)
 
except ValueError as e:
    print(e)
except:
    print("something is wrong")